Characterization-of-Habian-Motion/SPP Habian Motion.tex

```diff
@@ -0,0 +1,304 @@
+\usepackage{siunitx}
+% enable package for cross-referencing figures, sections, references etc.
+% how to use hyperref: http://www2.washjeff.edu/users/rhigginbottom/latex/resources/lecture09.pdf
+\usepackage{hyperref}
+% change text encoding to make it more crisp
+\usepackage[T1]{fontenc}
+% enable conditionals for help text
+\usepackage{etoolbox}
+
+% ----------------------------
+\newbool{showhelp}
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~
+% SHOW HELP TEXT IN THE PDF?
+\setbool{showhelp}{false}
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~
+% define conditional help text environment
+\newsavebox{\helpbox}
+\newenvironment{help}{
+  \ttfamily\footnotesize\sloppy
+  \begin{lrbox}{\helpbox}\begin{minipage}{\linewidth}
+  }{
+  \end{minipage}\end{lrbox}
+  \ifbool{showhelp}{
+    \fbox{\usebox{\helpbox}}
+  }{}
+}
+% ----------------------------
+
+% initialize nomenclature package
+\makenomenclature{}
+
+% set title. choose something as descriptive and precise as possible. Descriptive > sounding cool. remember this!
+\title{SPP: Characterization of Habian Motion}
+\author{
+  \begin{help}
+    List the authors of the proposal. The Champion should go first.
+    The \$~\$ markers tell \LaTeX{} to treat the text inside to be treated as a math expression. This way you can use operators like \textcaret{} to place characters as superscripts.
+    The \textbackslash{}thanks command puts the contents inside those brackets in a footnote at the bottom of the first page. Technically speaking, \textbackslash{}thanks is just a specially formatted footnote.
+    Read here for a more advanced options:  \url{http://tex.stackexchange.com/questions/826/symbols-instead-of-numbers-as-footnote-markers}
+  \end{help}
+  James~Emerson~Parkus$^{*\dagger}$%
+    \thanks{$^{*}$Project Champion}%
+    \thanks{$^{\dagger}$BS '22, Mechanical Engineering Technology \& Physics}
+
+  % the recommended order for symbolic footnotes is
+  %   (1) asterisk        *   *
+  %   (2) dagger          †   \dagger
+  %   (3) double dagger   ‡   \ddagger
+  %   (4) section symbol  §   \S
+  %   (5) paragraph       ¶   \P
+  %   et cetera. For higher counts, use 2x symbols (1)-(5) (i.e. (6) two asterisks **). Keep cycling through (1)-(5) using 3x, 4x, and so on.
+  %   Note that these symbol codes work in math mode and text mode.
+  %   There are ways to make LaTeX do this for you, but it is more advanced and not entirely necessary, especially for short author lists. Not worth the hassle, in my opinion.
+}
+% page header for pages other than cover page
+\markboth{Project Proposal Standard}%
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+  This project would be for the RIT SPEX High Altitude Balloon team. The goal is to create a map of the HAB condtions throughout
```

Comment:
If we want to get picky, I would consider what you have here to be an _executive summary_ more than an _abstract_. In the abstract I expected a little more technical detail on the implementation and scope of the project.